src/bin/wasmer.rs

```diff
@@ -132,6 +132,10 @@ struct Run {
     )]
     backend: Backend,
 
+    /// Invoke a specified function
```

Comment:
Maybe "specify the name of the start function" or "of the main function" of something like that?

Reply:
Is not a main function, but a raw function that you might want to invoke.